updatecli/updatecli.d/docker-ce.yaml

```
@@ -0,0 +1,15 @@
+---
+title: Bump docker-ce version
+sources:
+  default:
+    name: Get latest docker-ce version
+    kind: shell
+    spec:
+      command: bash ./updatecli/scripts/docker-ce-updates.sh
+targets:
+  default:
+    name: Update docker-ce version in common.yaml
+    kind: yaml
+    spec:
+      file: hieradata/common.yaml
+      key: "docker::version"
```

```suggestion
      key: "docker::version"

```
Nitpick: As per POSIX, all lines should end with an "endline" character (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline -> https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)

---

I agree